test/chimera/csv_to_xlsform_test.clj

```diff
@@ -72,7 +72,7 @@
 
 (facts "about merge-types-into-vector-schema"
        (fact "should handle types arg as a string"
-             (merge-types-into-vector-schema {:a {:type :old}} "new")
+             (merge-types-into-vector-schema {:a {:type :old}} "new" [["new"]])
```

Comment:
I wonder is the second argument to this function should be a list instead. If possible, maybe we should rename `"new"`(the type) to be something more descriptive and different from `"new"`(the column-value/select-one choice)

Reply:
seems this function is meant to receive any seq as a second argument. Maybe we just need an additional test case that shows that it can also receive a list/vector/seq as a second argument in addition to strings

Reply:
I have added a test for the select one